apps/test/unit/templates/SchoolAutocompleteDropdownTest.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../util/configuredChai';
+import SchoolAutocompleteDropdown from '@cdo/apps/templates/SchoolAutocompleteDropdown';
+
+describe('SchoolAutocompleteDropdown', () => {
+  it('renders VirtualizedSelect', () => {
+    const wrapper = shallow(
+      <SchoolAutocompleteDropdown
+        showErrorMsg={false}
+      />
+    );
+    expect(wrapper.find('VirtualizedSelect').exists()).to.be.true;
```

Comment:
:+1: Perfect, this is all I was hoping for - just a quick render test goes such a long way to catch syntax issues and such.